homeassistant/components/samsungtv/bridge.py

```diff
@@ -304,22 +316,22 @@ def try_connect(self):
 
         return RESULT_CANNOT_CONNECT
 
-    def device_info(self):
+    def device_info(self) -> Any:
```

Comment:
This can probably be typed to be a `dict` of some kind?

Reply:
Done